src/contracts/version/Registry.ts

```diff
@@ -37,6 +39,7 @@ export interface ExchangeAdapterInformation {
 
 export class Registry extends Contract {
   public static readonly abi = RegistryAbi;
+  public readonly MAX_REGISTERED_ENTITIES = 20;
```

Comment:
This value is public and might be nice to for an implementation to read from. So let's not give it a private const name and shout but instead call it `maxRegisteredEntities` only instead.